src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Rename/Renamer.cs

```diff
@@ -138,9 +138,9 @@ private Task<bool> RenameAsync(string oldFilePath, string newFilePath, bool isCa
 
                 DTE? dte = _dte.Value;
 
-                Assumes.NotNull(dte);
+                Assumes.Present(dte);
 
-                using var undo = UndoScope.Create(dte, renameOperationName);
+                using var _ = UndoScope.Create(dte, renameOperationName);
```

Comment:
why was this change made?

Reply:
The `undo` symbol is not used anywhere. My preference is to discard anything that's unused, so that I don't spend any time looking for usages of an unused symbol.